### ISSUE TITLE
Places information component: add accessibility

### ISSUE THREAD

**Issue by saumier** [culturecreates/footlight-app#265]

**Comment by AbhishekPAnil** [comment]
Estimate : 6
1.Add  the layout for venue accessibility and opening hours in the list item,check for responsiveness of the component.
2.Add the api integration for venue accessibility and opening hours.

**Comment by AbhishekPAnil** [comment]
@SuhailAliyar could you update the places listing api { https://staging.api.cms.footlight.io/places?excludeContainsPlace=true } with the respective address.Thank you.

**Comment by AbhishekPAnil** [comment]
@SuhailAliyar There was a mistake in the previous comment regarding the api.The api to be updated is {https://staging.api.cms.footlight.io/entities?query=&classes=Place}.Thank you.

**Comment by SuhailAliyar** [comment]
@AbhishekPAnil I've updated the entities listing API to return postal addresses, accessibility, and opening hours with places. 

Please find the changes in the staging server.

**Comment by saumier** [comment]
@troughc Please test in production. I saw that the postal code needs a space in front. Please take a look and enter your feedback for Abhishek.

**Comment by AbhishekPAnil** [comment]
@saumier @troughc ,Right now the way the address is displayed is :

First line => street adrress,
Second line=> addressLocality, addressRegion , postalCode

Please correct me if I am wrong.

**Comment by saumier** [comment]
@AbhishekPAnil You are correct. However please add a space to separate postal code from addressRegion (QC).

Current behaviour: Laval, QCH7L 2R3
Expected behaviour: Laval, QC H7L 2R3

**Comment by AbhishekPAnil** [comment]
> However please add a space to separate postal code from addressRegion (QC).

@troughc Space has been added before postal code and its updated in staging.

**Comment by troughc** [comment]
@AbhishekPAnil address looks good to me. 
Things to fix - screenshots at the bottom corresponding to each case listed below:
1. I added an opening hours to one of the places but it is not showing up in the component. The place is "Musée de la santé Armand-Frappier" (it is in STAGING) and the event where it doesn't show up is this one: http://staging.cms.footlight.io/dashboard/63247a1fea9d58004e3b03ec/events/add-event/644135a68edf85006402f6da . Screenshot below.
2. I did another test, I added accessiblity to the same place. When this is added the opening hours appear (see screenshot with accessiblity). These two items should not be linked. 
3. Also, if possible the opening hours should be a hyper link wiht the displayed text as "Opening hours" / "Heures d'ouverture". (like in the example at the top)
4. I added a french disambiguating description to the same place ("Musée de la santé Armand-Frappier"). I then deleted this place with the old disambiguating description from the event in case 1, and then added it back to display the french disambiguating description. This seemed to work (PART A); however, after I save and then go back into the event, the old (english) disambiguating description is being displayed (PART B).
5. I know the design has the icons centered but I don't like the way the Place icon and x (close) icons move down to be centered with the name and disambiguating description when it is very long. I am wondering if they should remain at the top (maybe top aligned to the name). I think it would look better.
6. There seems to be too much space between elements in the component: at the top of the place name; between place name and disambiguating description, between that and the next bits of info, between lines of text.
The last few screenshots are mockups of what i think this specific component should look like. 
I did note that when there is only an address and opening hours (no accessiblity information) the opening hours look kinda weird over on the right all by themselves (last screenshot). I don't have a good solution for this - unless we put opening hours under street address all the time. We can discuss this case before making any change @AbhishekPAnil 




### Case 1 and Case 4 part A
**screenshot with new disambiguating (french) description and NO opening hours and strange placements of icons**


![Image](https://user-images.githubusercontent.com/64489312/234693016-521311e9-02df-4706-b0cc-f0bfe11c9030.png)


### Case 2
**screenshot showing accessiblity and opening hours**

![Image](https://user-images.githubusercontent.com/64489312/234696908-7123e9a0-90c5-4256-9c24-1888e148729e.png)


### Case 4 Part B
**After saving and reopening event - screenshot with originally imported disambiguating (english) description and NO opening hours and strange placements of icons**



![Image](https://user-images.githubusercontent.com/64489312/234693181-99ab7174-9114-4346-8717-4c761ed6cbb7.png)





**After saving and reopening event - screenshot showing accessiblity, opening hours and old disambiguating description after a save an reopen**



![Image](https://user-images.githubusercontent.com/64489312/234698424-f4485ace-0f7c-4fc8-974c-d1ecc09b3d24.png)


### case 5 and 6
**how component could look with different elements**



![Image](https://user-images.githubusercontent.com/64489312/234700961-fb2d6ae2-0027-429f-9283-198ff64fca6e.png)



![Image](https://user-images.githubusercontent.com/64489312/234700995-e930d4a1-6924-4775-9821-99af957c90e5.png)



![Image](https://user-images.githubusercontent.com/64489312/234700839-0d3e6df6-3bb8-4f7e-b6cd-2360be4b9a64.png)


![Image](https://user-images.githubusercontent.com/64489312/234702676-e136b5ba-b358-41f3-bb0b-498bf3a493e3.png)
